src/praisonai/praisonai/_registry.py

```
@@ -0,0 +1,162 @@
+        self._items: Dict[str, Type[T]] = {}
+        self._lock = threading.Lock()
+        
+        # Load built-in plugins with error handling
+        if builtins:
+            for name, loader in builtins.items():
+                try:
+                    self._items[name] = loader()
+                except ImportError:
+                    # Built-in plugin dependencies not available, skip
+                    pass
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)